### PR TITLE
Add HTMX output

### LINKED ISSUE #3: Use htmx instead of json for RSVPs
Hi @zerok!

I'd like to change the RSVP endpoint to return a html fragment instead of json and get rid of the custom javascript in https://github.com/gograz/website and replace it with [htmx][]. htmx is small and well written javascript library which has zero dependencies and extends html tags in such a way that is enabling html to have modern features. htmx expects hypermedia (html fragments) in responses, instead of json.

In the website template it would look something like this:

```html
{{ if .Params.MeetupID }}
    <dl class="rsvps">
        <dt>Who's coming?</dt>
        <dd hx-get"https://api.gograz.org/meetup/{{ .Params.MeetupID }}/rsvps" hx-trigger="load delay:1s">
            Loading ...
        </dd>
    </dl>
{{ end }}
```

And in the API we would also have a template:

```html
{{ range . }}
    <a class="member" href="https://meetup.com/Graz-Open-Source-Meetup/members/{{ .ID }}" title="{{ .Name }}">
        {{ if .ThumbLink }}
             <img src="{{ .ThumbLink }}" alt="{{ .Name }}">
        {{ else }}
             <img src="/images/gopher.png" alt="{{ .Name }}">
        {{ end }}
    </a>
{{ else }}
    Be the first to RSVP!
{{ end }}
```

Would this be okay with you?

PS: I need an excuse to play around with htmx and do something useful, while doing so... 😉
PPS: I also prepare my talk for the next meetup!

[htmx]: https://htmx.org

### PR DESCRIPTION
**Pull request by zerok**
Fixes #3.

### CODE REVIEW

**Comment by m12r** [comment]
That looks pretty good, I'd say. Almost the same as I already did, but never quite finished... :) We can just use your's here! Sorry, for not finishing earlier.

**Comment by m12r** [comment]
Here's my WIP: https://github.com/m12r/gograz-meetup/commit/c5fcc4b684d16a004b1e9fa2ef7bd3a66483c882

**Review by m12r — APPROVED** [review]
Looks good to me! :+1:

File: templates/rsvps.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{ range .Yes }}
+    <a class="member" title="{{ .Name }}" href="https://www.meetup.com/Graz-Open-Source-Meetup/members/{{ .ID }}">
+        {{ if .ThumbLink }}
+        <img src="{{ .ThumbLink }}" alt="{{ .Name }}" />
+        {{ else }}
+        <img src="https://gograz.org/images/gopher.png" alt="{{ .Name }}" />
+        {{ end }}
+    </a>
+{{ else }}
+    Be the first to sign up!
```
Comment (m12r) [inline templates/rsvps.html:10]: Maybe you want to add a link here to meetup.com rsvp page.

Comment (zerok) [reply templates/rsvps.html]: Done :)

**Pull request merged**